src/server/services/LinkStatisticsService.ts

```diff
@@ -7,28 +7,37 @@ import { LinkStatisticsInterface } from '../../shared/interfaces/link-statistics
 import { UserRepositoryInterface } from '../repositories/interfaces/UserRepositoryInterface'
 import { NotFoundError } from '../util/error'
 import { logger } from '../config'
+import { DeviceCheckService } from './DeviceCheckService'
```

Comment:
we only need _DeviceCheckService_ for analytics, right? should we fold it into _LinkStatisticsService_?

Reply:
I am not so sure about that, as keeping `DeviceCheckService` separated does help with testing, and possibly code reuse in the future